packages/zoe/src/zoeService/installationStorage.js

```diff
@@ -9,11 +9,12 @@ import {
   InstallationShape,
   UnwrappedInstallationShape,
 } from '../typeGuards.js';
+import '../internal-types.js';
+import './internal-types.js';
```

Comment:
why was this necessary? we're trying to reduce the side-effect imports so I'm going to request changes for now

Reply:
Good question. I don't remember why I thought I needed it. After removing, `yarn lint` worked locally. Pushed. Now we'll see what CI has to say about it.

Reply:
CI is happy too. PTAL